README.md

```diff
@@ -5,13 +5,15 @@
 
 Gem used to send messages to Kafka in an easy way with an extra validation layer. It is a part of the [Karafka](https://github.com/karafka/karafka) ecosystem.
 
-WaterDrop is based on Zendesks [delivery_boy](https://github.com/zendesk/delivery_boy) gem.
+It:
 
-It is:
-
- - Thread safe
- - Supports sync and async producers
- - Working with 0.11+ Kafka
+ - Is thread safe
+ - Supports sync producing
+ - Supports async producing
+ - Supports buffering
+ - Supports producing messages to multiple clusters
+ - Supports multiple delivery policies
+ - Works with Kafka 1.0+ and Ruby 2.4+
```

Comment:
WDYT if we will add `No global state` here? I think it's an important sailing point for developers too